### PR TITLE
Desktop Version Portfolio Robin

### PR DESCRIPTION
**Pull request by robinbenitezmora**
This has been the chance to create the Desktop Version of my personal portfolio. In order to execute the Microverse requirements, the project was worked by means of media queries so that we have a responsive design useful for different screen sizes in diverse mobile, tablet, or desktop devices.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,38 +164,41 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
...
+        </div>
       </div>
     </section>
     <section class="formulary">
       <h2 class="formulary__title">Contact me</h2>
       <p class="formulary__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-      <img class="formulary__image" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <img class="formulary__image1" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <img class="formulary__image2" src="./images/contact-form-shapes-desktop.svg" alt="backgroung form">
       <form action="https://formspree.io/f/xeqnpkpe" method="post">
```
Comment (OmarRamoun) [inline index.html:202]: - [ ] [REQUIRED] Please, make sure that the form matches the figma design. In your website the form is aligned to the left. All you need to do is to put it in center.

image

File: index.html [hunk 2]
```diff
@@ -34,105 +34,117 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
     <section class="projects" id="projects">
       <section class="card1">
         <section class="card1__area">
-          <div>
-            <img class="card1__image" src="./images/card1.svg" alt="card1 image">
+          <div class="card1__pictures">
```
Comment (OmarRamoun) [inline index.html:37]: - [ ] [REQUIRED] Kindly, try to make the desktop version of the website similar to the figma design. For example: in the figma design, the text is centerd and the navigation menu is visible. In your design, the navigation menu doesn't show and the text is aligned to the left.

Your Design | Figma Design
------------|--------------
image | image

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi <STUDENT_MENTION> 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- :heavy_check_mark: Used BEM css classes :clap: (good one)
- The code is top-notch! 💯 and have a huge potential.

> **Reviewer Note**: The student code is so readable and clean. if he puts more effort on design, it is going to be even more awesome.

## Required Changes ♻️

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -152,50 +175,58 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
...
-    <section class="formulary">
+    <section class="formulary" id="contact">
       <h2 class="formulary__title">Contact me</h2>
-      <p class="formulary__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-      <img class="formulary__image" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <p class="formulary__paragraph">If you have an application you are interested in developing, a feature that you
+        need built or a project that needs coding. I’d love to help with it </p>
+      <img class="formulary__image1" src="./images/form-shapes-mobile@2x.svg" alt="backgroung form">
+      <img class="formulary__image2" src="./images/contact-form-shapes-desktop.svg" alt="backgroung form">
       <form action="https://formspree.io/f/xeqnpkpe" method="post">
```
Comment (HAMMAS-SALEEM) [inline index.html:214]: contact

- [ ] Please, make sure that the form matches the Figma design. On your website, the form is aligned to the left. All you need to do is to put it in the center.

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @robinbenitezmora :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @robinbenitezmora , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**